src/components/shared/button.json

```diff
@@ -52,6 +52,11 @@
         "quaternary",
         "quinary"
       ]
+    },
+    "theme": {
```

Comment:
Remark:
Buttons don't have a regular theme. It's kinda separate theme. 
<img width="613" alt="image" src="https://github.com/user-attachments/assets/0d898563-a38c-4d84-9a29-d3fbc7e88cdd">